docs/specs/slots.md

```diff
@@ -0,0 +1,27 @@
+# Slots
+
+Slots are used to replace property values during runtime. Properties that can be replaced will have an `sid` attribute that will correspond to the slot key entry. Players that support slots must also expose an API that applications can use to replace values during runtime.
+
+<h3 id="color-slot">Color Slot</h3>
+
+{schema_object:slots/color-slot}
+
+<h3 id="point-slot">Point Slot</h3>
+
+{schema_object:slots/point-slot}
+
+<h3 id="scale-slot">Scale Slot</h3>
```

Comment:
Point Slot and Scale Slot seem to be more context specific than the data focused names for Float Slot and Color Slot. They could theoretically both fall under something like Vec2 Slot. However, this is a naming issue alone, and it might be best to keep it consistent for now with how existing exporters have introduced the idea of slots.

Context: https://github.com/bodymovin/bodymovin-extension/blob/4f1de8ebfe5beaebc4c7d54fbbdf363ca685478c/bundle/jsx/utils/essentialPropertiesHelper.jsx#L15

Reply:
From a JSON schema perspective, integral enums would be easier to work with. Regex can do some of the compound string processing, though arbitrary lengths may push beyond the limits.

We also have some more complex object types, such as image asset which may not fit as cleanly into compound string type system, though we could still still special case those.